stage-1/create-release-pr/action.yml

```diff
@@ -0,0 +1,67 @@
+name: Create pull request
+description: Create the release PR
+# NOTE: Needs GITHUB_TOKEN env variable (and HEAD_REF)
+
+inputs:
+  pr-label:
+    description: The label to add to the PR
+    required: false
+    default: 'release'
+  message:
+    description: Commit message
+    required: false
+    default: 'Prepare release ${VERSION}'
+  commit-all:
+    description: If true, stage all files before comitting
+    required: false
+    default: false
+  # registry:
+  #   description: What package registry to print in the PR if publishing package, e.g. 'PyPI'
+  #   required: false
```

Comment:
Might be good to have this for testing.

Reply:
Note in this action it could only change what is printed in the PR, it wouldn't actually affect what registry is published to. So I removed it to try and keep things more simple

Reply:
Was thinking about the PYPI test repository.

Reply:
Not a biggie for us at the moment though as Cylc8 is still in alpha and Cylc7 is not on PyPi.